@kiva/kv-components/vue/KvSwitch.vue

```diff
@@ -78,14 +80,7 @@ export default {
 			default: false,
 		},
 		/**
-		 * Value of the input if this component is used within a form submission
```

Comment:
Removing `value` prop from checkbox and switch since I don't think it would ever be used in a Vue environment.